guides/v2.3/graphql/queries/customer-cart.md

```diff
@@ -0,0 +1,79 @@
+---
+group: graphql
+title: customerCart query
+---
+
+The `customerCart` query returns the active cart for the logged-in customer. If the cart does not exist, the query creates one. The customer's authorization token must be specified in the header of the call.
+
+The `customerCart` query differs from the `cart` query in that it must be run on behalf of a logged-in customer. You cannot run this query on behalf of a guest.
+
```

Comment:
You can run the query for the guest, but it will throw an exception. In other words, guest won't see any results in the response.